manifests/resource/up.pp

```diff
@@ -11,27 +13,31 @@
   # create metadata on device, except if resource seems already initalized.
   # drbd is very tenacious about asking for aproval if there is data on the
   # volume already.
-  exec { "initialize DRBD metadata for ${name}":
-    command => "yes yes | drbdadm create-md ${name}",
-    onlyif  => "test -e ${disk}",
-    unless  => "drbdadm dump-md ${name} || (drbdadm cstate ${name} | egrep -q '^(Sync|Connected|WFConnection|StandAlone|Verify)')",
-    before  => Service['drbd'],
-    require => [
-      Exec['modprobe drbd'],
-      Concat["/etc/drbd.d/${name}.res"],
+  if $initialize {
+    exec { "initialize DRBD metadata for ${name}":
+      command => "yes yes | drbdadm create-md ${name}",
```

Comment:
Why is `yes` listed twice?

Reply:
I just changed the indenting of that lines, but I just tested it. It is because of this two interactive questions:  
```
root@test01:~# drbdadm create-md data

Valid meta data seems to be in place.
Do you really want to overwrite?
[need to type 'yes' to confirm] yes

md_offset 0
al_offset 4096
bm_offset 36864

Found some data

 ==> This might destroy existing data! <==

Do you want to proceed?
[need to type 'yes' to confirm] 

```